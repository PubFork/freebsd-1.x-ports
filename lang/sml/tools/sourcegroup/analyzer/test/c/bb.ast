[closing c/bb.sml]
val it =
  SeqDec
    [MarkDec
       (FctDec
          [MarkFctb
             (Fctb
                {def=FctFct
                       {body=MarkStr
                               (StructStr
                                  (SeqDec [MarkDec (OpenDec [[prim?]],27,33)]),
                                20,37),constraint=NONE,
                        params=[(SOME prim?,MarkSig (VarSig prim?,15,16))]},
                 name=prim?},10,37)],2,37)] : System.Compile.ast
- 