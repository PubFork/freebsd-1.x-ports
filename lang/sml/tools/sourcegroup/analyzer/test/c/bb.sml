functor G (D:A) = struct open D end
