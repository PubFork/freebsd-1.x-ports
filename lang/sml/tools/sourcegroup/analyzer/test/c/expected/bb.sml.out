source sml "c/bb.sml"
  import signature A 
  export functor G ;

